vllm/config.py

```diff
@@ -170,14 +170,17 @@ class SchedulerConfig:
             a single iteration.
         max_num_seqs: Maximum number of sequences to be processed in a single
             iteration.
+        max_prompt_len: Maximum length of input prompt.
```

Comment:
The actual limit should not be `max_prompt_len`. Actually, it should be `max_sequence_len` that is applied to the sum of the prompt length and the generated output length.  Can you add `max_sequence_len` check also for the sum? Thanks!